community/kernel/src/main/java/org/neo4j/helpers/Args.java

```diff
@@ -252,7 +252,15 @@ public long getDuration( String key, long defaultValueInMillis)
     public Boolean getBoolean( String key, Boolean defaultValue )
     {
         String value = getSingleOptionOrNull( key );
-        return value != null ? Boolean.parseBoolean( value ) : defaultValue;
+
+        // Apparently this condition must be split like this, instead of as an elvis operator,
+        // because defaultValue will, in that case, be evaluated as a primitive boolean and
+        // a NullPointerException will insue. Odd.
+        if ( value != null )
+        {
+            return Boolean.parseBoolean( value );
+        }
+        return defaultValue;
```

Comment:
That's happens because of autoboxing of `defaultValue` to boolean because Boolean.parseBoolean will return primitive type.
if expression will be changed to `value != null ? Boolean.valueOf( value) : defaultValue` ternary operator can be used.